Userland/Libraries/LibC/netdb.cpp

```diff
@@ -829,4 +829,27 @@ int getnameinfo(const struct sockaddr* __restrict addr, socklen_t addrlen, char*
 
     return 0;
 }
+
+void herror(char const* s)
+{
+    int saved_herrno = h_errno;
+    dbgln("herror(): {}: {}", s, hstrerror(saved_herrno));
+    warnln("{}: {}", s, hstrerror(saved_herrno));
```

Comment:
Should be make sure to preserve `h_errno` s so that calling `herror()` doesn't clobber it?

Reply:
Actually, it is unnecessary. Will fix.